src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -689,14 +683,23 @@ public static string RetrieveValueFromConnStr(string connStr, string[] keywords)
 
         public class TraceEventListener : EventListener
         {
+            private IList<EventSource> _eventSources = new List<EventSource>();
```

Comment:
This TraceEventListener class is also used for EventSourceTest. Is there any reason this is changed here?

Reply:
The reason for the change - is the class can only enable the trace collection, but it doesn't disable it when it is no longer needed.
The result of a "non-disabled" trace collection is not visible unless there is only one trace listener all over the tests. But if at some point other listeners do appear - this one would interfere with their trace collecting.

Reply:
Thank you for the explanation. This sounds reasonable.